README.md

```diff
@@ -18,6 +18,13 @@ The plugin examines `composer` type repositories. For any that contain an additi
 during package discovery and download operations, validating that the repository and package are not being tampered
 with.
 
+In accordance with the [TUF specification](https://github.com/theupdateframework/specification/blob/v1.0.9/tuf-spec.md#5-detailed-workflows),
+projects using this plugin must supply a set of trusted keys for each repository they want to protect with TUF. Each
+TUF-protected repository should provide a JSON file named for the repository URL, with non-alphanumeric characters
+replaced by a single period. For example, if the repository URL is `https://repo.example.net:1234/packages`, then the
+key file should be called `https.repo.example.net.1234.packages.json`. All key files must be stored in a directory
```

Comment:
I don't think we can just replace everything with `.`

`site.coop.info/packages`  and `site.coop/info/packages` would both be converted to `site.coop.info.packages` 
so probably need to either have folder everything before first `/` or replace `/` with something else. maybe `-`

Reply:
Yeah, you're right...I suspect we are going to use a somewhat different scheme (discussing with @davidstrauss below, so marking this as resolved).